src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -50,7 +67,98 @@ public String getColor() {
         return color;
     }
 
-    public List<Section> getSections() {
-        return sections;
+    public List<Station> getStations() {
+        return this.sections.getStations();
+    }
+
+    public void removeSection(Section section) {
+        sections.remove(section);
+    }
+
+    public boolean hasSection(Section section) {
+        return sections.contains(section);
+    }
+
+    public void addSection(Section section) {
+        if (!hasSection(section)) {
+            sections.add(section);
+        }
+
+        if (!section.equalsLine(this)) {
+            section.toLine(this);
+        }
+    }
+
+    public void addLineStation(Station upStation, Station downStation, int distance) {
+        if (sections.isEmptyStation()) {
+            Section.create(this, upStation, downStation, Distance.valueOf(distance));
+            return;
+        }
+
+        validateForAdded(upStation, downStation);
+
+        sections.updateSection(upStation, downStation, distance);
+        addSection(Section.create(this, upStation, downStation, Distance.valueOf(distance)));
+    }
+
+    public void removeLineStation(Station station) {
+
+        Optional<Section> upLineStation = sections.findByUpStation(station);
+        Optional<Section> downLineStation = sections.findByDownStation(station);
+
+        validateForRemove(upLineStation, downLineStation);
+
+        if (upLineStation.isPresent() && downLineStation.isPresent()) {
+            addLineStationByRemove(upLineStation.get(), downLineStation.get());
+        }
+
+        upLineStation.ifPresent(it -> sections.remove(it));
+        downLineStation.ifPresent(it -> sections.remove(it));
```

Comment:
해당 로직은 Line 보다는 Sections에 적합한 책임으로 보이는데요! 
책임을 Sections로 이전해보면 어떨까요 ?

```suggestion
        sections.removeStation(station)
```